api/test/e2e/route_with_management_fileds_test.go

```diff
@@ -333,6 +333,17 @@ func TestRoute_search_by_label(t *testing.T) {
 			ExpectBody:   "\"total_size\":2",
 			Sleep:        sleepTime,
 		},
+		{
+			Desc:         "search the route by label (combination)",
+			Object:       ManagerApiExpect(t),
+			Path:         "/apisix/admin/routes",
+			Query:        "label=build:16,build:17",
+			Method:       http.MethodGet,
+			Headers:      map[string]string{"Authorization": token},
+			ExpectStatus: http.StatusOK,
+			ExpectBody:   "\"total_size\":2",
```

Comment:
just to confirm, is this all contents in response?

Reply:
Yes.